neurots/generate/orientations.py

```diff
@@ -56,6 +62,7 @@ def __init__(self, soma, parameters, distributions, context, rng):
 
         self._orientations = {}
         self._modes = self._collect_mode_methods()
+        self.pia_direction = np.array([0.0, 1.0, 0.0])
```

Comment:
take this from the defined parameter?

Reply:
it is not in the parameter, unless we add it. In the other file, this direction is quite harcoded, I wonder if we should really hardcode it, or leave it free all the way?

Reply:
I've added it in utils.py as a variable we can import